docs/resources/cc_bandwidth_package.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Cloud Connect (CC)"
+---
+
+# huaweicloud_cc_bandwidth_package
+
+Manages a bandwidth package resource of Cloud Connect within HuaweiCloud.  
+
+## Example Usage
+
+```hcl
+variable "project_id" {}
+
+resource "huaweicloud_cc_bandwidth_package" "test" {
+  name           = "demo"
+  local_area_id  = "Chinese-Mainland"
+  remote_area_id = "Chinese-Mainland"
+  interflow_mode = "Area"
+  charge_mode    = "bandwidth"
+  billing_mode   = 3
+  bandwidth      = 5
+  project_id     = var.project_id
+  description    = "This is a demo"
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) The bandwidth package name.  
+  The name can contain a maximum of 64 characters.
+
+* `local_area_id` - (Required, String, ForceNew) The local area ID.  
+  Valid values are **Chinese-Mainland**, **Asia-Pacific**, **Africa**, **Western-Latin-America**,
+   **Eastern-Latin-America** and **Northern-Latin-America**.
+
+  Changing this parameter will create a new resource.
+
+* `remote_area_id` - (Required, String, ForceNew) The remote area ID.  
+  Valid values are **Chinese-Mainland**, **Asia-Pacific**, **Africa**, **Western-Latin-America**,
+   **Eastern-Latin-America** and **Northern-Latin-America**.
+
+  Changing this parameter will create a new resource.
+
+* `charge_mode` - (Required, String, ForceNew) Billing option of the bandwidth package.  
+  Valid value is **bandwidth**.
+
+  Changing this parameter will create a new resource.
+
+* `billing_mode` - (Required, String) Billing mode of the bandwidth package.  
+  The options are as follows:
+    + **3**: pay-per-use for the Chinese Mainland website.
+    + **4**: pay-per-use for the International website.
+    + **5**: 95th percentile bandwidth billing for the Chinese Mainland website.
+    + **6**: 95th percentile bandwidth billing for the International website.
+
+  -> This argument can only be modified to **5** and **6**.
+
+* `bandwidth` - (Required, Int) Bandwidth in the bandwidth package.  
+
+* `project_id` - (Required, String, ForceNew) Project ID.
```

Comment:
the CC service is global, why should we input the project_id and what's the valid value of this parameter?

Reply:
change to optional